jgrapht-core/src/main/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTree.java

```diff
@@ -45,8 +45,7 @@
  * algorithm for subset disjoint cycles by Ahuja et al. That is, the algorithm may run in
  * exponential time. This algorithm is implemented in two different version: a local search and a
  * tabu search. In both cases we have to find the best neighbor of the current capacitated spanning
- * tree. For further information about finding such an improving neighbor
- * {@link ImprovementGraph} @see ImprovementGraph
+ * tree. 
```

Comment:
Why do we delete the last sentence?

Reply:
The class is private, so the link is not usable. You get a warning for an undefined reference in the javadocs.